libs/xp-systems/src/xp-systems.class.ts

```diff
@@ -187,6 +187,20 @@ export class XpSystems {
     return rankGain;
   }
 
+  getPrestige(totalLevel: number): number {
+    return Math.floor(Math.max(totalLevel - 1, 0) / 500);
+  }
+
+  getInnerLevel(totallevel: number): number {
+    return ((totallevel - 1) % 500) + 1;
+  }
+
+  getPrestigeIcon(prestige: number): string {
+    return prestige <= 5
+      ? `assets/images/prestige/prestige${prestige}.svg`
+      : 'assets/images/prestige/max_level.svg';
+  }
+
```

Comment:
This code lives in a shared library so would rather not have frontend-specific paths in here - I'd just stick this method inside of the level indicator component